modules/logging-loki-gui-install.adoc

```diff
@@ -35,14 +33,132 @@ include::snippets/logging-stable-updates-snip.adoc[]
+
+.. Select *Enable Operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+You must select this option to ensure that cluster monitoring
+scrapes the `openshift-logging` namespace.
+
+.. Select *stable-5.y* as the *Update Channel*.
+
+.. Select an *Approval Strategy*.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+. Go to the *Operators* -> *Installed Operators* page. Click the *All instances* tab.
+
+. From the *Create new* drop-down list, select *LokiStack*.
+
+. Select *YAML view*, and then use the following template to create a `LokiStack` CR:
++
+--
+.Example `LokiStack` CR
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging # <2>
+spec:
+  size: 1x.small # <3>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: "2022-06-01"
+    secret:
+      name: logging-loki-s3 # <4>
+      type: s3 # <5>
+      credentialMode: # <6>
+  storageClassName: <storage_class_name> # <7>
+  tenants:
+    mode: openshift-logging # <8>
+----
+<1> Use the name `logging-loki`.
+<2> You must specify the `openshift-logging` namespace.
+<3> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<4> Specify the name of your log store secret.
+<5> Specify the corresponding storage type.
+<6> Optional field, logging 5.9 and later. Supported user configured values are as follows: static is the default authentication mode available for all supported object storage types using credentials stored in a Secret. token for short-lived tokens retrieved from a credential source. In this mode the static configuration does not contain credentials needed for the object storage. Instead, they are generated during runtime using a service, which allows for shorter-lived credentials and much more granular control. This authentication mode is not supported for all object storage types. token-cco is the default value when Loki is running on managed STS mode and using CCO on STS/WIF clusters.
+<7> Specify the name of a storage class for temporary storage. For best performance, specify a storage class that allocates block storage. Available storage classes for your cluster can be listed by using the `oc get storageclasses` command.
+<8> LokiStack defaults to running in multi-tenant mode, which cannot be modified. One tenant is provided for each log type: audit, infrastructure, and application logs. This enables access control for individual users and user groups to different log streams.
+--
+
```

Comment:
The numbering restarts in the next step and I believe a + here would resolve that:
```suggestion
+
```